src/projecteurapp.cc

```diff
@@ -268,7 +268,7 @@ ProjecteurApplication::ProjecteurApplication(int &argc, char **argv, const Optio
 
   // Open local server for local IPC commands, e.g. from other command line instances
   QLocalServer::removeServer(localServerName());
-  if (m_localServer->listen(localServerName()))
+  if (m_localServer && m_localServer->listen(localServerName()))
```

Comment:
`m_localServer` is instantiated in the constructor. If that fails (memory allocation error) the program will fail long before reaching this point.

Reply:
Yes, I saw that, but, just to turn off warning I have introduced extra check for `m_localServer`. However, it is not necessary. Will remove it in next commit.